Document changing the sandbox image in CR (#33660)
kubernetes/website: content/en/docs/setup/production-environment/container-runtimes.md

When using kubeadm, manually configure the
[cgroup driver for kubelet](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node).

> #### Overriding the sandbox (pause) image

Reviewer: ```suggestion
#### Overriding the sandbox (pause) image {#override-pause-image-containerd}
```

For CRI-O, the CRI socket is `/var/run/crio/crio.sock` by default.

> #### Overriding the sandbox (pause) image

Reviewer: ```suggestion
#### Overriding the sandbox (pause) image {#override-pause-image-cri-o}
```

Check the systemd unit named `cri-docker.socket` to find out the path to the CRI
socket.

> ## Overriding the sandbox (pause) image

Author: I wasn't sure if this should be it's own section or sub headings under the other CR docs, felt nice to keep this separate.

Reviewer 2: I think these should be separate sub sections under the individual CRs.

For `cri-dockerd`, the CRI socket is `/run/cri-dockerd.sock` by default.

> #### Overriding the sandbox (pause) image

Reviewer: ```suggestion
#### Overriding the sandbox (pause) image {#override-pause-image-cri-dockerd}
```

To overwrite the sandbox (pause) image the `cri-dockerd` adapter
[accepts a flag](https://github.com/Mirantis/cri-dockerd)

> to configure this `--pod-infra-container-image`.

Reviewer 2: we can just duplicate this section under `### Mirantis Container Runtime`.

Reviewer 2: other than this detail the change LGTM. thanks!

/lgtm

> to configure this `--pod-infra-container-image`.

Reviewer: (nit)
```suggestion
to configure this. The command line argument to use is `--pod-infra-container-image`.
```

> To overwrite the sandbox (pause) image the `cri-dockerd` adapter
> [accepts a flag](https://github.com/Mirantis/cri-dockerd/blob/master/src/config/options.go)
> to configure this `--pod-infra-container-image`.

Reviewer 2: ```suggestion
To overwrite the sandbox (pause) image the [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter
accepts the `--pod-infra-container-image` flag.
```

instead of linking to the source code, suggesting that we link to the main `cri-dockerd` page with hopes that its documentation will enumerate flags eventually.

> To overwrite the sandbox (pause) image the `cri-dockerd` adapter
> [accepts a flag](https://github.com/Mirantis/cri-dockerd)
> you can configure this. The command line argument to use is `--pod-infra-container-image`.

Reviewer: This language isn't idiomatic English. How about:
```suggestion
The `cri-dockerd` adapter accepts a command line argument for
specifying which container image to use as the Pod infrastructure container (“pause image”).
The command line argument to use is `--pod-infra-container-image`.
```

Check the systemd unit named `cri-docker.socket` to find out the path to the CRI
socket.

> #### Overriding the sandbox (pause) image

Reviewer: ```suggestion
#### Overriding the sandbox (pause) image {#override-pause-image-cri-dockerd-mcr}
```

To overwrite the sandbox (pause) image the `cri-dockerd` adapter
[accepts a flag](https://github.com/Mirantis/cri-dockerd)

> to configure this `--pod-infra-container-image`.

Reviewer: (nit)
```suggestion
to configure this. The command line argument to use is `--pod-infra-container-image`.
```

> you can configure this. The command line argument to use is `--pod-infra-container-image`.

Reviewer: This should be the same as whatever you do to fix the previous text.

Check the systemd unit named `cri-docker.socket` to find out the path to the CRI
socket.

> 

Reviewer 2: here we can just copy the same `#### Overriding the sandbox (pause) image` from the section from `### Docker Engine`

### Docker Engine and Mirantis Container Runtime

{{< note >}}
If using the Docker Engine these instructions assume that you are using the
[`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter to integrate
Docker Engine with Kubernetes.
{{< /note >}}

> 

Reviewer 2: i think we can omit this addition given both sections `### Docker Engine` and `### Mirantis Container Runtime` already mention `cri-dockerd`?
